Update log-analytics-alerts-actions.md

JakobGSvendsen opened the pull request:
when using new alerts there is new format for the webhook data. added .Data to a few files

azurecla commented:
Hi __@ashishmansukhani__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@ashishmansukhani : Thanks for your contribution to the Azure documentation! The author, @masnider, has been notified to review your proposed change.

masnider commented:
Hi @ashishmansukhani, , these changes look good. I #signoff, let me know if there is other steps I need to take.  Thanks!

ashishmansukhani commented:
Hi @masnider , I don't have sufficient permissions to merge this. Will @azurecla do it automatically?

masnider commented:
Well, I thought they would :) @tysonn - what do we need to do here?

tysonn commented:
@masnider - Type #sign-off as the author, and the ready-to-merge label will be assigned. That lets us know it's ready to be merged.

masnider commented:
@tysonn hm, I was told in a separate conversation with @rwike77 that didn't work (and I guess I thought I had when I did signoff all one word above). Whatever! Glad to do whatever is necessary. Will do in the future.

rwike77 commented:
@tysonn  Yes, I did tell @masnider that in a separate mail.  I must've misread the instructions, my mistake.

JakobGSvendsen committed:
Update log-analytics-alerts-actions.md

when using new alerts there is new format for the webhook data. added .Data to a few files
